nimble/host/src/ble_hs_stop.c

```diff
@@ -21,9 +21,12 @@
 #include "sysinit/sysinit.h"
 #include "syscfg/syscfg.h"
 #include "ble_hs_priv.h"
+#include "nimble/nimble_npl.h"
+#ifndef MYNEWT
+#include "nimble/nimble_port.h"
+#endif
 
-static ble_npl_event_fn ble_hs_stop_term_event_cb;
-static struct ble_npl_event ble_hs_stop_term_ev;
+#define BLE_HOST_STOP_TIMEOUT_MS (2000)
```

Comment:
It would be better if this timeout were were configurable.

Reply:
agree